### DISCUSSION TITLE
For some reason my waybar no longer has the "pill" look for the groups of modules

### DISCUSSION THREAD

**Discussion by fsckr** [prasanthrangan/hyprdots#2046] [Q&A]
The only changes I have made is the size of waybar.  Here is the waybar that I am using:

`1|40|top|( hyprland/workspaces ) ( custom/power custom/cliphist custom/wbar custom/theme custom/wallchange )|( custom/cava custom/updates clock )|( pulseaudio pulseaudio#microphone tray custom/keybindhint custom/cliphist custom/power )`

Here is a screenshot of what it looks like:
![image](https://github.com/user-attachments/assets/6ed74628-2e77-4559-859c-c0a9b6fe9df6)

Any ideas??

**Comment by kRHYME7** [comment]
what theme?

**Reply by fsckr** [reply]
The theme in the screenshot was I believe Blood Red, but it happens on every theme.  They all used to have rounded corners that give it the "pill" look.  I have tried cycling through the waybar modes and they are all that way.

**Reply by fsckr** [reply]
Well come to find out it is only happening to the theme bad blood.  The others seem to be fine.  I will have to try and figure out why.  What is weird is it was fine the other day.  My apologies if I wasted anyone's time.  Hope everyone has a good holiday!
